content/content.js

```diff
@@ -142,10 +145,12 @@ function loadForemHTML(forems) {
   if (forems.length === 0 && !document.getElementById('forem-gettingstarted')) {
     const gettingStartedDiv = document.createElement('DIV');
     gettingStartedDiv.id = 'forem-gettingstarted';
-    gettingStartedDiv.innerHTML = "👈 Use the + button to save a forem in your switcher"
-    newStyles.innerHTML = newStyles.innerHTML + "#forem-gettingstarted { position: fixed; left: 70px; bottom: 10px; background: black; color: white;padding: 10px 15px; border-radius: 8px;}";
+    gettingStartedDiv.innerHTML =
+      '👈 Use the + button to save a forem in your switcher';
+    newStyles.innerHTML =
+      newStyles.innerHTML +
+      '#forem-gettingstarted { position: fixed; left: 70px; bottom: 10px; background: black; color: white;padding: 10px 15px; border-radius: 8px; z-index:999}';
```

Comment:
I just added z-index here I think prettier (in git hook) is formatting the code.

Reply:
Nice stuff. This is me using `--no-verify` before. I'm sorry @nickytonline 😭